### Fixed

- The automatic annotation process failed for tasks from cloud data
  (<https://github.com/cvat-ai/cvat/pull/8199>)
